srcpkgs/qemu-user-static/template

```diff
@@ -12,73 +12,15 @@ homepage="https://www.qemu.org"
 distfiles="https://wiki.qemu.org/download/qemu-${version}.tar.bz2"
 checksum=8314b6e5fcc7bf9fa3915d504de6586a69cba30ffa27cbe9ba85d2cb9987fb3a
 
-_fmts="aarch64 alpha arm armeb cris i386 m68k microblaze mips mipsel mips64 mips64el ppc ppc64 ppc64abi32 ppc64le s390x sh4 sh4eb sparc sparc32plus sparc64 x86_64"
```

Comment:
The new list is missing ppc64abi32, sparc, sparc64, cris

Reply:
I have accounted for sparc and sparc64. Someone needs to write a patch for QEMU to add ppcel (@q66 ?) and cris:

https://gitlab.com/qemu-project/qemu/-/blob/master/scripts/qemu-binfmt-conf.sh

https://wiki.qemu.org/Contribute/SubmitAPatch

Reply:
On second thought, do we even want to have cris? It was ripped out of Linux in 4.16 (2018). And the existing ppcel patch just needs to be updated to take into account our use of the linked script.

Reply:
I'm okay with it being removed, just mention in the commit message please :)

Reply:
alright mentioned